docs/taskruns.md

```diff
@@ -77,8 +77,8 @@ A `TaskRun` definition supports the following fields:
   - [`workspaces`](#specifying-workspaces) - Specifies the physical volumes to use for the
     [`Workspaces`](workspaces.md#using-workspaces-in-tasks) declared by a `Task`.
   - [`debug`](#debugging-a-taskrun)- Specifies any breakpoints and debugging configuration for the `Task` execution.
-  - [`stepOverrides`](#overriding-task-steps-and-sidecars) - Specifies configuration to use to override the `Task`'s `Step`s.
-  - [`sidecarOverrides`](#overriding-task-steps-and-sidecars) - Specifies configuration to use to override the `Task`'s `Sidecar`s.
+  - [`stepSpecs`](#configuring-task-steps-and-sidecars-in-a-taskrun) - Specifies configuration to use to override the `Task`'s `Step`s.
+  - [`sidecarSpecs`](#configuring-task-steps-and-sidecars-in-a-taskrun) - Specifies configuration to use to override the `Task`'s `Sidecar`s.
```

Comment:
It would be nice to have two difference examples for each stepSpecs and sidecarSpecs here instead of going to the same link

Reply:
There are actually two different examples in tabs. I'm not sure how to link to a tab. I tried using the `id` of rendered HTML of `tabs-0-1-tab`, but that only links to the tab and doesn't open it. I'm also not sure how stable that `id` is given it's rendered by a macro.